components/st-microelectronics/landpatterns.stanza

```diff
@@ -0,0 +1,183 @@
+#use-added-syntax(jitx)
+defpackage ocdb/st-microelectronics/landpatterns:
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/land-patterns
+  import ocdb/defaults
+  import ocdb/box-symbol
+  import ocdb/design-vars
+  import ocdb/tolerance
+
+;================================================================
+; Missing Datasheets
+;================================================================
+
+; public pcb-landpattern LFBGA448:
+;   fatal("unimplemented")
+
+; public pcb-landpattern TFBGA257:
+;   fatal("unimplemented")
+
+; public pcb-landpattern TFBGA361:
+;   fatal("unimplemented")
+
+; public pcb-landpattern EWLCSP49:
+;   fatal("unimplemented") ; TODO find datasheet
+
+; public pcb-landpattern EWLCSP66:
+;   fatal("unimplemented") ; TODO find datasheet
+
+; public pcb-landpattern WLCSP59:
+;   fatal("unimplemented") ; TODO find datasheet
+
+;================================================================
+; Weird Edge Cases
+;================================================================
+; public pcb-landpattern TFBGA240+25:
+;   fatal("unimplemented") ; todo: handle TFBGA240
+
+; public pcb-landpattern UFBGA176+25:
```

Comment:
Is `+` ok in a `pcb-landpattern`?

Reply:
Technically yes, but there's a good question over whether we should allow it in our naming conventions. This is how those land pattern/packages are named in the data sheets.